### DISCUSSION TITLE
How do I see / access the network traffic flow just as I can in mitmweb within my Python console through mitm python API?

### DISCUSSION THREAD

**Discussion by Avnsx** [mitmproxy/mitmproxy#5184] [Q&A]
The docs are not giving any examples or anything on that, it just says there's a python api and explains classes and stuff, which doesn't directly help me

**Comment by mhils** [comment]
I'm not sure if I understand what your question is? Did you see https://docs.mitmproxy.org/dev/addons-overview/?
